src/app/modules/portal-shared/card/card.component.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../../assets/scss/varibles.scss";
+
+.c-title {
+  white-space: nowrap;
+  width: 200px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  height: 30px;
+}
+
+.mat-card img {
+  // object-fit: cover; /*this makes the image in src fit to the size specified below*/
+  // width: 100%; /* Here you can use wherever you want to specify the width and also the height of the <img>*/
+  height: 120%;
```

Comment:
why is ite given  greater than 100%,  Please use VH to give height

Reply:
changed